components/time/src/zone/mod.rs

```diff
@@ -190,11 +190,25 @@ impl TimeZone {
 
 /// This module exists so we can cleanly reexport TimeZoneVariantULE from the provider module, whilst retaining a public stable TimeZoneVariant type.
 pub(crate) mod ule {
-    /// A time zone variant, such as Standard Time, or Daylight/Summer Time.
+    /// An indication of a period when a time zone maps to a specific UTC offset.
     ///
-    /// This should not generally be constructed by client code. Instead, use
+    /// Although the variants are named `Standard` and `Daylight`, what they actually indicate
+    /// is whether the period has a lower offset or a higher offset from UTC. Their behavior may
```

Comment:
I don't want to say this because I want to have the freedom to model Morocco as CET or add a third variant.

Reply:
But this doesn't mention Morocco anymore. I'm just writing down what rearguard DST is defined to mean.

Reply:
This higher/lower classification falls apart when we add a third variant. The way this should be described is that "standard" is the offset that is considered the standard offset, and daylight is a potential higher offset. Ramadan will be a potential lower offset.